packages/server/src/controllers/Controller.js

```diff
@@ -111,6 +112,68 @@ export class Controller {
   }
 
   setupAction(type, name, handler, authorize) {
+    if (!isFunction(handler)) {
+      const setupHandlerFromParam = ({
+        handler,
+        action,
+        authorize,
+        parameters,
+        returns,
+        scope,
+        transacted
+      }) => {
+        if (action !== undefined) {
```

Comment:
I think all of these checks can just check for them being truthy (`if (action)`) instead of against `undefined`? Or do we need to be able to set them to null explicitly?